apps/site/app/[locale]/next-data/blog-data/[category]/[page]/route.ts

```diff
@@ -20,12 +20,11 @@ export const GET = async (_: Request, props: StaticParams) => {
 
   const requestedPage = Number(params.page);
 
-  const data =
-    requestedPage >= 1
-      ? // This allows us to blindly get all blog posts from a given category
-        // if the page number is 0 or something smaller than 1
-        providePaginatedBlogPosts(params.category, requestedPage)
-      : provideBlogPosts(params.category);
+  const data = await (requestedPage >= 1
```

Comment:
Can you separate the method call and await to somewhere else? Like add the `await` directly on `return Response.json(await data, ...)`

Reply:
change reverted, no longer needed (thanks to work done on the adapter 🙂) 🥳